enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/components/SdkIframeEmbedRoute.tsx

```diff
@@ -84,11 +89,31 @@ export const SdkIframeEmbedRoute = () => {
     >
       <Box h="100vh" bg={theme?.colors?.background}>
         <SdkIframeEmbedView settings={embedSettings} />
+        <AnalyticsReporter usageAnalytics={usageAnalytics} />
       </Box>
     </ComponentProvider>
   );
 };
 
+interface AnalyticsReporterProps {
+  usageAnalytics: {
+    usage: EmbeddedAnalyticsJsEventSchema;
+    embedHostUrl: string;
+  } | null;
+}
+function AnalyticsReporter({ usageAnalytics }: AnalyticsReporterProps) {
+  const instanceUrl = useSetting("site-url");
+  useEffect(() => {
+    const isEmbeddedAnalyticsJsPreview =
+      usageAnalytics && usageAnalytics.embedHostUrl?.startsWith(instanceUrl);
```

Comment:
This is to ensure if users run Metabase in a subpath it would still work.

e.g. users could run metabase at example.com/metabase

and embed Metabase at example.com/embed

in this scenario, it shouldn't be considered as the same site.